lib/private/machines/create-manager.js

```diff
@@ -125,21 +125,11 @@ module.exports = {
     var mongoUrl = _clientConfig.url;
     _clientConfig = _.omit(_clientConfig, ['url', 'user', 'password', 'host', 'port', 'database']);
 
-    // Use unified topology. MongoDB node maintainers recommends this to be enabled
-    // https://github.com/mongodb/node-mongodb-native/releases/tag/v3.2.1
-    // Use new url parser to remove warnings
-    _clientConfig = Object.assign({
```

Comment:
Hey @mikermcneil this line has to go because the configs we are setting here has been deprecated and produces warnings.

Reply:
I wonder if this could be related to the `?` error @eashaw mentioned (no real reason why I'm suggesting that, other than at a glance, I'm noticing `useNewUrlParser`, so maybe related?)